webapp/security/models.py

```diff
@@ -147,6 +147,17 @@ class Status(Base):
         "pending",
     ]
 
+    all_statuses = [
+        "released",
+        "DNE",
+        "needed",
+        "not-affected",
+        "deferred",
+        "needs-triage",
+        "ignored",
+        "pending",
+    ]
```

Comment:
Where are we using this? the variable `all_statuses` is not being mentioned anywhere else in `models.py`.

As an aside, I was wondering if we are interested in converting `active_statuses` as an `Enum`?

Reply:
`all_statuses` is used in view.py:375 to default to `all_statuses` if `Any` is selected in the status drop-down. I chose to put it in the model, since it belongs to the `status` entity.

Not sure what you mean by converting `active statuses` as an `Enum`.

Reply:
Well, you don't have to do this, but I was suggesting to get `all_statuses` list from the table (as query) as I believe these are stored in a table line 166 of this `models.py`?

Reply:
Ok.. I found out how to fetch the enum of statuses from the database. In a way I am not keen on adding an extra database query, but it is probably nice to have only one place that stores all the statuses. I will make the change shortly.

Reply:
Done.

Reply:
> Ok.. I found out how to fetch the enum of statuses from the database. In a way I am not keen on adding an extra database query, but it is probably nice to have only one place that stores all the statuses. I will make the change shortly.

Yeah, I would not have asked if I didn't see the other `Enum`. Because it is true that initially we did it without `Enums` but if we are using them in one place, and then using static statuses somewhere else, this could lead us to different versions of these statuses.

Nice!